tests/baselines/reference/checkJsxChildrenProperty14.errors.txt

```diff
@@ -48,7 +49,8 @@ tests/cases/conformance/jsx/file.tsx(42,11): error TS2322: Type '{ children: Ele
     // Error
     let k5 = <SingleChildComp a={10} b="hi"><></><Button /><AnotherButton /></SingleChildComp>;
               ~~~~~~~~~~~~~~~
-!!! error TS2322: Type '{ children: Element[]; a: number; b: string; }' is not assignable to type 'SingleChildProp'.
-!!! error TS2322:   Types of property 'children' are incompatible.
-!!! error TS2322:     Type 'Element[]' is not assignable to type 'Element'.
-!!! error TS2322:       Property 'type' is missing in type 'Element[]'.
\ No newline at end of file
+!!! error TS2322: Type '{ children: Element[]; a: number; b: string; }' is not assignable to type 'IntrinsicAttributes & SingleChildProp'.
+!!! error TS2322:   Type '{ children: Element[]; a: number; b: string; }' is not assignable to type 'SingleChildProp'.

```

Comment:
I think it was special-case code that got rid of this line in the first place. Is it possible that it just needs to be moved or updated to work again?

Reply:
As I mentioned in person, this is a result of bringing SFC checking back into line and doing the right thing - there was no "special case" removing the jsx types from the errors; they just straight up weren't being checked against!